.golangci.yml

```diff
@@ -169,3 +169,13 @@ issues:
   exclude-use-default: false
   max-issues-per-linter: 0
   max-same-issues: 0
+  exclude-rules:
```

Comment:
Probably we need to disable one of two linters if they are conflicted

Reply:
Yep, don't actually sure that `sloppyReassign` is any useful, have disabled it.